run/main.go

```diff
@@ -74,11 +74,19 @@ signalLoop:
 			fmt.Println(" <INTERRUPT>")
 			log.Warning("main: program was interrupted, shutting down.")
 
+			forceCnt := 5
```

Comment:
Move variable declaration to L80 (before the `for`). Since it's only used inside the go() there's no need to create a closure and let the variable escape to the heap.